tvb_framework/tvb/interfaces/web/controllers/base_controller.py

```diff
@@ -98,10 +98,12 @@ def __init__(self):
             IntrospectionRegistry.SIIBRA_CREATOR_MODULE,
             IntrospectionRegistry.SIIBRA_CREATOR_CLASS
         )
-        siibra_link = self.get_url_adapter(siibra_algo.fk_category, siibra_algo.id)
-        self.connectivity_submenu.append(dict(title='Siibra Connectivity', link=siibra_link,
-                                              subsection=WebStructure.SUB_SECTION_SIIBRA,
-                                              description='Import connectivities from EBRAINS KG using siibra'))
+        if os.environ.get('HBP_AUTH_TOKEN'):
```

Comment:
Could this check also be placed in siibra_creator before the initialisation of atlas and parcellation lists ?

Reply:
I don't think so.. if we place it there, without the token, the atlas and parcellation lists would just not be initialized, but everything (every class) inside the siibra_creator module would still be created

Reply:
Creating the classes should not be a problem, even with empty select lists.
I am just worried that with the initialisation/communication to siibra in the absence of the token, the builds take a long time to run...